frontend/app/equity/dividend_computations/[id]/confirm/page.tsx

```
@@ -0,0 +1,292 @@
+
+function ConfirmDividendComputationContent() {
+  const params = useParams();
+  const router = useRouter();
+  const [computation, setComputation] = useState(mockComputation);
+  const [isLoading, setIsLoading] = useState(false);
+  const totalRecipients = computation.outputs.length;
+  const totalShares = computation.outputs.reduce((sum, output) => sum + (output.number_of_shares || 0), 0);
+
+  useEffect(() => {
+    // Load computation data from localStorage for demo
+    const computationId = parseInt(params.id as string);
```

ESLint

[error] Missing radix parameter.

(radix)